daemon/network.go

```diff
@@ -262,6 +263,11 @@ func (daemon *Daemon) createNetwork(create types.NetworkCreateRequest, id string
 		driver = c.Config().Daemon.DefaultDriver
 	}
 
+	if driver == "overlay" &&
+		(daemon.clusterProvider == nil || !daemon.clusterProvider.IsManager()) {
+		return nil, errorutils.ErrNotSwarmManager
```

Comment:
Will this be a problem if someone uses an external k/v store for overlay networks?